src/Sokoban.hs

```diff
@@ -36,6 +37,9 @@ data Cell = MkCell {
 }
     deriving (Eq, Show)
 
+-- Level number and its environment
+type Level = (Int, Environment)
```

Comment:
Do you anticipate needing to associate any other data with a level? If so, maybe a new `data` type class may be easier to work with than the `type` synonym.

Reply:
Not sure, but yeah, I refactored the code to use `data` instead.